client/helperFunctions.js

```diff
@@ -1,12 +1,8 @@
-const findManagers = (users, products) => {
-	return users.reduce((acc, user) => {
-		products.forEach(product => {
-			if (user.id === product.managerId && !acc.includes(user)) {
-				acc.push(user);
-			}
-		});
-		return acc;
-	}, []);
-};
+const findManagers = (users, products) =>
```

Comment:
You're using `reduce` in a non-functional way. Thats fine, but non-ideal in the react ecosystem. This never mutates an array.